mod/tagrm.php

```diff
@@ -63,12 +61,7 @@ function update_tags($item_id, $tags){
 	}
 
 	$tag_str = implode(',',$old_tags);
-	if(!empty($tag_str)) {
-		Item::update(['tag' => $tag_str], ['id' => $item_id]);
-	}
-	else {
-		Term::deleteByItemId($item_id);
-	}
+	Term::insertFromTagFieldByItemId($item_id, $tag_str);
```

Comment:
Is the `insertFromTagFieldByItemId` `$tags` parameter supposed to be a string?

Reply:
You're right. I guess I should stop afterwards for today. I'm too tired to concentrate. This would be easier if an doxygen header :sweat_smile:  in insertFromTagFieldByItemId

Reply:
Once we move to minimum PHP 7 compatibility starting with the `2019.03-dev` version, we'll be able to type-hint on strings and integers on top of arrays and classes.